tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/CompilerServiceBenchmarks.fs

```diff
@@ -105,7 +105,8 @@ type CompilerServiceBenchmarks() =
             | Some _ -> configOpt
             | None ->
                 let checker = FSharpChecker.Create(projectCacheSize = 200)
-                let source = FSharpSourceText.From(File.OpenRead("""..\..\..\..\..\..\..\..\..\src\Compiler\Checking\CheckExpressions.fs"""), Encoding.Default, FSharpSourceHashAlgorithm.Sha1, true)
+                let path = Path.Combine("..", "..", "..", "..", "..", "..", "..", "..", "..", "src", "Compiler", "Checking", "CheckExpressions.fs")
```

Comment:
Hah yeah that's crazy. Probably an even better way would be to follow what's in BackgroundCompilerBenchmark [here](https://github.com/dotnet/fsharp/blob/f0eccb0e362f01d3ae2cf6edae3485123b0dc3f3/tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/BackgroundCompilerBenchmarks.fs#L108).

Reply:
Oh, yes. Thanks.